cmd/cloudx/client/handler_test.go

```diff
@@ -58,6 +58,222 @@ func TestCommandHelper(t *testing.T) {
 		return &notYetLoggedIn
 	}
 
+	t.Run("func=SetDefaultProject", func(t *testing.T) {
```

Comment:
We can probably parallelize these tests (with `t.Parallel()`), in all places.

Reply:
:+1: